django_server/.env.sample

```
@@ -0,0 +1,80 @@
+# Django Configuration
+SECRET_KEY=your-secret-key-here-change-in-production
+DEBUG=True
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DEBUG key should go before the SECRET_KEY key

(UnorderedKey)